src/main/scala/io/gatling/sbt/settings/gatling/RecoverEnterprisePluginException.scala

```diff
@@ -32,10 +32,11 @@ class RecoverEnterprisePluginException(config: Configuration) {
   protected def recoverEnterprisePluginException[U](logger: ManagedLogger): PartialFunction[Throwable, Try[U]] = {
     case e: UnsupportedJavaVersionException =>
       logger.error(s"""${e.getMessage}
-                      |In order to target the supported Java bytecode version, please use the following sbt settings:
-                      |scalacOptions += "-target:${e.supportedVersion}"
+                      |In order to target the supported Java version, please use the following sbt settings:
+                      |scalacOptions ++= Seq("-target:${e.supportedVersion}", "-release", "${e.supportedVersion}")
```

Comment:
My understanding is that using both `target` and `release` for scalac is the equivalent to using `release` for javac. (FYI, in Scala 3, `release` alone is enough, and this behavior has just been backported to Scala 2.13.x for the next release by this pull request: scala/scala#9982.)

Reply:
Also: in the other plugins, we only provide guidance for javac. Probably not a big deal at the moment because Kotlin and Scala still default to emitting Java 8 bytecode.

Here we provide help for javac and scalac, because we only really support SBT for Scala for Scala. In practice the scalac setting is unlikely to be useful though, unless the user already manually set the `target` to 18 or later...

Reply:
As scalac may be used to compile java files, the `source` may be different from the `target`.
`release` is a shorthand for both to the same value.